code/modules/admin/admin_holder.dm

```diff
@@ -44,6 +44,7 @@
 	var/datum/centcomviewer/centcomviewer = null
 	var/datum/bioeffectmanager/bioeffectmanager = null
 	var/datum/abilitymanager/abilitymanager = null
+	var/datum/antagonist_panel/antagonist_panel = null
```

Comment:
do we need a per-admin one instead of a global one?

Reply:
Yes, the reason as to why is outlined in `antagonist_panel.dm`, however, briefly, it is so that only the required tabs have their data generated, as opposed to generating all tab data. This necessitates the current tab being passed to the backend and stored in per-user datums.

Overall, this results in less data being generated and passed to the UI, increasing performance.